.pylintrc

```diff
@@ -21,3 +21,4 @@ disable=
 
 max-args=10
 max-line-length=130
+max-positional-arguments=15
```

Comment:
Can we just get rid of pylint? lol
I hate a linter that we never actually listen to.